output.tf

```diff
@@ -1,3 +1,4 @@
 output "web_instance_ip" {
     value = aws_instance.public.public_ip
 }
+
```

Comment:
This extra newline is not needed, please remove it

Reply:
Updated removed the extra line